lambda/main.tf

```diff
@@ -85,7 +85,7 @@ resource "aws_lambda_permission" "lambda_permissions" {
 
 resource "aws_iam_role" "lambda_iam_role" {
   assume_role_policy = templatefile("${path.module}/templates/lambda_assume_role.json.tpl", {})
-  name               = "${var.function_name}Role"
+  name               = "${var.function_name}-role"
```

Comment:
Does this need `role` appended? We know it's a role as it's in IAM and the ARN will contain `:role/...`